### PR TITLE
site: Adds first submenu for Community

### PR DESCRIPTION
**Pull request by codefromthecrypt**
This adds a submenu for community, filing GH and the history into it.
The same surgery should work for future submenus, for example,
compatibility and docs.

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
https://deploy-preview-557--wazero.netlify.app/

File: site/layouts/partials/menu.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+              <li><a href="{{ .URL }}">{{ .Name }}</a></li>
+            {{ end }}
+          </ul>
+        </ul>
+      {{ end }}
+    {{ end }}
+  </ul>
+
+  <!-- This is the theme default layouts/partials/menu.html -->
+  <ul class="menu__inner menu__inner--mobile">
```
Comment (codefromthecrypt) [inline site/layouts/partials/menu.html:25]: there was no CSS for mobile submenus, so I decided to let this ride. another way could be to concatenate when children exist (ex About/GitHub About/History, etc)

**Comment by anuraaga** [comment]
Looks like `Community` doesn't show on mobile just want to confirm that is intentional

image

**Comment by codefromthecrypt** [comment]
wrt mobile not dropdown, well not so much intentional, but much more work as the underlying theme didn't have CSS etc for mobile. I can change to prefix it with a TODO for someone to make CSS

**Comment by codefromthecrypt** [comment]
only issue with prefixing is that it will eventually become long. anyway this isn't worse for mobile, as before it truncated anything with children also. it fixes desktop though.

**Review by anuraaga — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
I did the inlining for now as at least that's easy. since mobile already makes a drop-down it wasn't so bad on my phone. good catch @anuraaga

**Pull request merged**